Kitodo/src/main/java/de/sub/goobi/forms/AktuelleSchritteForm.java

```diff
@@ -952,35 +956,19 @@ public void setHideCorrectionTasks(boolean hideCorrectionTasks) {
         this.hideCorrectionTasks = hideCorrectionTasks;
     }
 
-    /**
-     * Return the id of the current task.
-     *
-     * @return int stepId
-     */
-    public int getStepId() {
-        return this.stepId;
-    }
-
-    /**
-     * Set the id of the current task.
-     *
-     * @param stepId
-     *            as int
-     */
-    public void setStepId(int stepId) {
-        this.stepId = stepId;
-    }
-
     /**
      * Method being used as viewAction for AktuelleSchritteForm.
+     *
+     * @param id
+     *            ID of the step to load
      */
-    public void loadMyStep() {
+    public void loadMyStep(int id) {
         try {
-            if (!Objects.equals(this.stepId, null)) {
-                setMySchritt(this.serviceManager.getTaskService().getById(this.stepId));
+            if (!Objects.equals(id, null)) {
```

Comment:
An `int` type can or should never be null, maybe you want to use an `Integer` type which can be null. On other places you are comparing this with value `0`, so maybe only a wrong written condition?